lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -669,8 +665,7 @@ bool check_ue_context_state(amf_ue_ngap_id_t ue_id,
   }
 
   n2cause_e ue_context_rel_cause;
-  rc = amf_get_ue_context_rel_cause(ue_id, &ue_context_rel_cause);
-  if (rc != RETURNok) {
+  if (amf_get_ue_context_rel_cause(ue_id, &ue_context_rel_cause) != RETURNok) {
```

Comment:
Not related to your changes, since you are refactoring you can remove "n2cause_e expected_ue_context_rel_cause" passed to this function as it is being over written at 667